client/lib/abtest/active-tests.js

```diff
@@ -104,5 +104,13 @@ module.exports = {
 		defaultVariation: 'original',
 		allowAnyLocale: true,
 		allowExistingUsers: true
+	},
+	nudgeStatsGoogleAnalytics: {
+		datestamp: '20160320',
```

Comment:
I think we should backdate it a bit.
New user is not going to be power user and we want to target the power ones